demo/actions/keyword_handlers.py

```diff
@@ -0,0 +1,5 @@
+from response.core.models.incident import Incident
```

Comment:
There is a reference to this file, as it's being imported but as the file is empty, the file wasn't in GIT. I've added some default import statements that we'll need so the file isn't empty by default.